mlflow/store/sqlalchemy_store.py

```diff
@@ -2,27 +2,92 @@
 import uuid
 
 from mlflow.entities.lifecycle_stage import LifecycleStage
+from mlflow.store.dbmodels.db_types import DB_ENGINES, MYSQL
 from mlflow.store.dbmodels.models import Base, SqlExperiment, SqlRun, SqlMetric, SqlParam, SqlTag
 from mlflow.entities import RunStatus, SourceType
 from mlflow.store.abstract_store import AbstractStore
 from mlflow.entities import ViewType
 from mlflow.exceptions import MlflowException
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, RESOURCE_ALREADY_EXISTS, \
     INVALID_STATE, RESOURCE_DOES_NOT_EXIST
+from mlflow.utils.file_utils import build_path, exists, mkdir
 from mlflow.utils.mlflow_tags import MLFLOW_PARENT_RUN_ID, MLFLOW_RUN_NAME
 from mlflow.utils.search_utils import does_run_match_clause
 
 
+DEFAULT_ARTIFACT_ROOT = "artifact_root"
+
+
+class DbHelper:
+    def __init__(self, db_type, session):
+        if db_type not in DB_ENGINES:
```

Comment:
Given DbHelper is only used during default experiment creation, this seems like a slightly odd place to put this. Why not in the `__init__` of `SqlAlchemyStore`?

Reply:
This helper is needed for setting `sql_mode` and this setting for different databases. Wanted to avoid adding all these cases in `SqlAlchemyStore.__init__`.

Will rename this to `DatabaseHelper`